lib/plugins/TaskRunner/WordPressApp.js

```diff
@@ -105,7 +104,8 @@ class WordPressApp extends LAMPApp {
 
     // The boilerplate is significantly more simple.
     this.script = this.script.concat([
-      `if [ !  -a "/var/www/html/wp-config.php" ] ; then`,
+      'mkdir -p /var/www/html',
```

Comment:
At least for my testing, it was problematic to write to the wp-config.php if html dir didn't exist yet.